docker/tds/docker-compose.yml

```diff
@@ -20,25 +35,26 @@ services:
     ports:
       - "32843:8080"
     env_file: tds-docker.env
-  exam_mysql:
-    image: mysql
-    ports:
-      - "32777:3306"
-    volumes_from:
-      - exam_mysql_data
-    env_file: tds-docker.env
-  exam_mysql_data:
-    image: mysql
-    entrypoint: /bin/bash
   exam:
     image: fwsbac/tds-exam-service
     ports:
       - "80:8080"
     depends_on:
-      - assessment
-      - session
-      - student
-      - config
-      - exam_mysql
-    env_file:
-      - ./tds-docker.env
+      configuration-service:
+        condition: service_healthy
+      assessment:
+        condition: service_started
+      session:
+        condition: service_started
+      student:
+        condition: service_started
+      config:
+        condition: service_started
+    environment:
+      - CONFIG_SERVICE_URL=http://configuration-service:8888
+      - CONFIG_SERVICE_ENABLED=true
+      - SPRING_PROFILES_ACTIVE=local-docker
```

Comment:
This configures the exam service to pull configuration values from the configuration-service.  It will make a request to http://configuration-service:8888/exam-service/local-docker on startup and fast-fail if it doesn't get a viable response.